scripts/testnet/sovereignBridge/config/header-verifier.snippets.sh

```diff
@@ -0,0 +1,69 @@
+HEADER_VERIFIER_ADDRESS=$(mxpy data load --partition=${CHAIN_ID} --key=address-header-verifier-contract)
+
+deployHeaderVerifierContract() {
+    manualUpdateConfigFile #update config file
+
+    echo "Deploying Header Verifier contract on main chain..."
+
+    BLS_PUB_KEYS=$(python3 $SCRIPT_PATH/pyScripts/read_bls_keys.py)
+
+    local OUTFILE="${OUTFILE_PATH}/deploy-header-verifier.interaction.json"
+    mxpy contract deploy \
+        --bytecode=$(eval echo ${HEADER_VERIFIER_WASM}) \
+        --pem=${WALLET} \
+        --proxy=${PROXY} \
+        --chain=${CHAIN_ID} \
+        --gas-limit=200000000 \
+        --arguments ${BLS_PUB_KEYS} \
+        --outfile=${OUTFILE} \
+        --recall-nonce \
+        --wait-result \
+        --send || return
+
+    printTxStatus ${OUTFILE} || return
+
+    local ADDRESS=$(mxpy data parse --file=${OUTFILE}  --expression="data['contractAddress']")
+    mxpy data store --partition=${CHAIN_ID} --key=address-header-verifier-contract --value=${ADDRESS}
+    HEADER_VERIFIER_ADDRESS=$(mxpy data load --partition=${CHAIN_ID} --key=address-header-verifier-contract)
+    echo -e "Header Verifier contract: ${ADDRESS}\n"
+}
+
+upgradeHeaderVerifierContract() {
+    manualUpdateConfigFile #update config file
+
+    echo "Upgrading Header Verifier contract on main chain..."
+
+    local OUTFILE="${OUTFILE_PATH}/upgrade-header-verifier.interaction.json"
+    mxpy contract upgrade ${HEADER_VERIFIER_ADDRESS} \
```

Comment:
I feel like we re-write this function of upgrading contract in many other places;
Is there any way to make a base func out of it?

Reply:
I would have to send so many parameters to a base function (message, outfile, address, wasm, proxy, chain, gas). Also arguments can be sent if we need in the future and that would be variable, etc... 
Also, probably, these upgrade funcs will be deleted in the future from deploy scrips when we'll have factory/config SC.